data/json/items/ammo/exodii.json

```
@@ -223,6 +223,24 @@
     "effects": [ "NEVER_MISFIRES" ],
     "casing": "caotel_cell_spent"
   },
+  {
+    "id": "khuunaofaai_ammo_firework",
+    "type": "AMMO",
+    "range": 100,
+    "name": { "str": "firework payload" },
```

What tests are yelling about is that your ammo for a gun do not contain any word from the gun, so it is impossible to relate one to another
```suggestion
    "name": { "str": "khuunäöfaai firework payload" },
```

---

thanks!

---

That didn't seem to fix it because I get the same errors, actually.

---

Apparently it doesn't recognize the word "cannon" so I'm fixing that